gitkraken.sh

```diff
@@ -1,3 +1,3 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
my arch system doesn't use that. Like @MarcinWieczorek said, /bin/sh ( /usr/bin/sh ) is the default.